_ont/ont-huawei-ma5671a.md

```diff
@@ -49,17 +49,14 @@ Configuration: asc0=0 115200 8-N-1
 
 - [Root](/ont-huawei-ma5671a-root)
 
-##  Disabling Dying Gasp
-```sh
-set nDyingGaspEnable disable
```

Comment:
Should be replaced with `fw_setenv nDyingGaspEnable 0`

Reply:
Should be moved into the rooted section IMHO